salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -36,12 +37,16 @@ Install containerd:
 Create containerd service drop-in:
   file.managed:
     - name: /etc/systemd/system/containerd.service.d/50-metalk8s.conf
-    - source: salt://{{ slspath }}/files/50-metalk8s.conf
+    - source: salt://{{ slspath }}/files/50-metalk8s.conf.j2
+    - template: jinja
     - user: root
     - group: root
     - mode: 0644
     - makedirs: true
     - dir_mode: 0755
+    - context:
+      environment: >-
+        NO_PROXY=127.0.0.1,localhost,{{ networks.values() | join(",") }}
```

Comment:
shouldnt we set `.svc,.default,.local,.cluster.local` too ?

Reply:
Sure

Reply:
hmm yes i guess it is